uni/lib/controller/networking/network_router.dart

```diff
@@ -175,4 +175,18 @@ class NetworkRouter {
   static List<String> getBaseUrlsFromSession(Session session) {
     return NetworkRouter.getBaseUrls(session.faculties);
   }
+
+  /// Makes an HTTP request to terminate the session in Sigarra.
+  static Future killAuthentication(List<String> faculties) async {
+    final url =
+    '${NetworkRouter.getBaseUrl(faculties[0])}vld_validacao.sair';
+    final response = await http.
+      get(url.toUri()).timeout(const Duration(seconds: loginRequestTimeout));
+    if (response.statusCode == 200) {
+      Logger().i("Logout Successful");
+    }else{
+      Logger().i("Logout Failed");
+    }
+    return response;
+  }
```

Comment:
Loop the faculties list to terminate session in each faculty.
Also, is a nice idea to include the faculty value in the Logger output